bon-macros/src/builder/builder_gen/setters/mod.rs

```diff
@@ -166,6 +179,29 @@ impl<'a> SettersCtx<'a> {
         Ok([self.setter_method(some_fn), self.setter_method(option_fn)].concat())
     }
 
+    fn getter_method(&self, item: Item) -> TokenStream {
```

Comment:
I'd like to avoid mixing code for setters and getters, especially because this file resides in a module called `setters`. I belive it should be in a totally separate `getters` module. Just make sure setters and getters are generated adjacently in the resulting impl block.